src/components/newComponent/pos/Cart.tsx

```
@@ -0,0 +1,394 @@
+                    size="sm"
+                    onClick={() => updateQuantity(item.product.id, item.quantity + 1)}
+                    className="p-1 h-8 w-8"
+                  >
+                    <Plus className="w-3 h-3" />
+                  </Button>
+                </div>
+
+                {/* Remise */}
+                <div className="mb-3">
+                  <label className="block text-xs text-gray-500 mb-1">Remise (XAF)</label>
+                  <Input
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)